apps/cowswap-frontend/public/emergency.js

```diff
@@ -7,6 +7,44 @@ if (window.location.host === 'barn.cowswap.exchange') {
   window.location.href = 'https://barn.cow.fi'
 }
 
+/**
+ * Remove old versions of the local storage atom stores
+ * We rely on the fact that store names are in the format {name}Atom:v{version}
+ * Since outdated versions of the stores are not used anymore, we should remove them to not exceed the storage limit
+ */
+;(function () {
+  const atomStoreRegex = /^(.+Atom):v(\d{1,3})$/
+
+  // Find the latest version of each store
+  const storePerVersion = Object.keys(localStorage)
+    // Take only the atom stores with versions
+    .reduce((acc, key) => {
+      const match = key.match(atomStoreRegex)
+
+      if (!match) return acc
+
+      const [, name, version] = match
+      const versionNum = +version
+
+      // Find the latest version of the store
+      if (!acc[name] || acc[name] < versionNum) {
+        acc[name] = versionNum
+      }
+
+      return acc
+    }, {})
+
+  // Remove all the old versions
+  Object.keys(storePerVersion).forEach((name) => {
+    const version = storePerVersion[name]
+
+    for (let i = 0; i < version; i++) {
+      localStorage.removeItem(`${name}:v${i}`)
```

Comment:
ubernit, you will attempt to delete keys that don't exist, but is fine...

Reply:
There is no exception, I've tried.
And it also doesn't have any impact on performance